src/command_modules/azure-cli-acr/README.rst

```diff
@@ -86,39 +86,39 @@ Update a container registry
         az acr update: Update a container registry.
```

Comment:
For consistency with other CLI modules you may want to wrap this in a generic update (which adds the --add, --set and --remove flags). `network nic update` is a good example of an update command that exposes the generic parameters and convenience parameters (which are the ones you already expose).